package.json

```diff
@@ -25,7 +25,9 @@
     "db:update-client": "npx prisma generate",
     "spec:generate": "cross-env NODE_ENV=development ts-node scripts/generateOpenApi.ts && npm run spec:validate",
     "spec:validate": "cross-env NODE_ENV=development ts-node scripts/validateOpenApi.ts",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "prepush": "npm run lint && npm run audit",
```

Comment:
`lint` should not be a part of prepush. It's perfectly fine to push messy, horrible, broken code to your own branch and fix it later. By making it harder to do so, we are incentivizing developers to postpone pushing code until it's absolutely ready, which is a _very_ undesireable behaviour.

What we want to avoid is bad code being merged to the main. Which is already enforced by having linting as a part of CI workflow.

`audit` shouldn't be a part of prepush either. It doesn't matter that we have vulnerable dependency until we are merging something - having this executed as a part of ci.yml workflow should be perfectly sufficient, so let's add it as a step there.